pkg/kubernetes/scripts/projects.js

```diff
@@ -676,6 +876,189 @@
         }
     ])
 
+    .factory('memberActions', [
+        "kubeMethods",
+        function(methods) {
+            function removeUserFromGroup(user, group) {
+                var userName = toName(user);
+                var users = group.users || [];
+                var index = users.indexOf(userName);
+                if (index >= 0)
+                    users.splice(index, 1);
+                var patchData = { "users": users };
+                return methods.patch(group, patchData);
+            }
+            function addUserToGroup(user, group) {
+                var userName = toName(user);
+                var users = group.users || [];
+                var index = users.indexOf(userName);
+                if (index == -1)
+                    users.push(userName);
+                var patchData = { "users": users };
+                return methods.patch(group, patchData);
+            }
+            return {
+                removeUserFromGroup: removeUserFromGroup,
+                addUserToGroup: addUserToGroup,
+            };
+        }
+    ])
+
+    .controller('UserChangeCtrl', [
+        '$q',
+        '$scope',
+        'kubeSelect',
+        'kubeLoader',
+        "kubeMethods",
+        'projectData',
+        'projectPolicy',
+        '$location',
+        'memberActions',
+        "fields",
+        function($q, $scope, kselect, loader, methods, projectData, projectPolicy, $location, memberActions, fields) {
+            function getPolicyBinding(namespace){
+                return kselect().kind("PolicyBinding").namespace(namespace).name(":default");
+            }
+
+
+            function getMembers() {
+                var members = [];
+                var groups = getGroups();
+                var projects = getProjects();
+                angular.forEach(groups, function(group) {
+                    members.push(group);
+                });
+                angular.forEach(projects, function(project) {
+                    members.push(project);
+                });
+                return members;
+            }
+            function getProjects() {
+                return kselect().kind("Project");
+            }
+            function getGroups() {
+                return kselect().kind("Group");
+            }
+            $scope.select = {
```

Comment:
Can we call this something else, the naming is confusing and makes me think of our use of kubeSelect.